.circleci/config.yml

```diff
@@ -25,14 +25,13 @@ jobs:
               command: ./scripts/build.sh
 
           - run:
-              name: Run unit tests
+              name: FHIR model validation
               command: |
                 source ./venv/bin/activate
-                pip install -r dev-requirements.txt
-                pytest -s tests
-
+                fhirutil validate $(pwd)/site_root/ig.ini --clear_output --publisher_opts='-tx n/a'
```

Comment:
You're right but I think it's like this now because Docker containers require absolute paths for bind mounted host volumes. I can change this so that the user doesn't have to worry about supplying an absolute path. I'll make an issue on ncpi-fhir-utility

Reply:
Oh looks like I already do this in ncpi-fhir-utility: https://github.com/ncpi-fhir/ncpi-fhir-utility/blob/0eb7d2769070d4e2524781072ec6b79643421453/ncpi_fhir_utility/app.py#L62